.dev/reverse-imports-no-repos

```diff
@@ -0,0 +1,15 @@
+package,maintainer
+adaptalint,conway.max1@gmail.com
```

Comment:
definitely also want to .gitignore this one to avoid posting emails (even thought they're easily available from CRAN) -- again keeping it in the PR for now for viz